frontend/core-ui/src/modules/logs/utils/logRecordTableUtils.ts

```
@@ -0,0 +1 @@
+export const generateListQueryParams = (params: any) => {};
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)